.travis.yml

```diff
@@ -1,8 +1,13 @@
-language: ruby
+language: node_js
+
+node_js:
+- 7
```

Comment:
Should we use 8 as it's LTS?

Reply:
Sure. I'm using 8.7 on my local machine. Shall we add a card to try and update `govuk-frontend` too at some point?

Reply:
Probably worth doing, yep.